### PR TITLE
Small bug fixes

### PR DESCRIPTION
**Pull request by wli3**
- Localization for CommonOptions
- Remove accidental debug line

**Customer scenario**

N/A

**Bugs this fixes**

[CLI #6291](https://github.com/dotnet/cli/issues/6291)

**Workarounds, if any**

N/A

**Risk**

Common command opinion help message will not be localized in the future

**Performance impact**

Low

**Root cause analysis**

N/A

**How was the bug found?**

Code review

---

**Customer scenario**

Type _dotnet restore -h_ and user will see extra line of "-h" on top of help command

**Bugs this fixes**

[CLI #6360](https://github.com/dotnet/cli/issues/6360)

**Workarounds, if any**

N/A

**Risk**

The customer will be confused by the extra line of help message.

**Performance impact**

Low

**Root cause analysis**

N/A

**How was the bug found?**

Sayed's script to print out all customer facing texts.

### CODE REVIEW

**Comment by wli3** [comment]
@livarcocc  we need this change go into release/2.0.0 right? especially for extra "-h" line

File: src/dotnet/CommonLocalizableStrings.cs [hunk 1]
```diff
@@ -185,5 +185,16 @@ internal class CommonLocalizableStrings
         public const string ProjectNotCompatibleWithFrameworks = "Project `{0}` cannot be added due to incompatible targeted frameworks between the two projects. Please review the project you are trying to add and verify that is compatible with the following targets:";
         public const string ProjectDoesNotTargetFramework = "Project `{0}` does not target framework `{1}`.";
         public const string ProjectCouldNotBeEvaluated = "Project `{0}` could not be evaluated. Evaluation failed with following error:\n{1}";
+
+        /// common options
```
Comment (livarcocc) [inline src/dotnet/CommonLocalizableStrings.cs]: I don't think we need this comment.

File: src/dotnet/CommonLocalizableStrings.cs [hunk 2]
```diff
@@ -185,5 +185,16 @@ internal class CommonLocalizableStrings
...
+        public const string VerbosityOptionDescription = "Set the verbosity level of the command. Allowed values are q[uiet], m[inimal], n[ormal], d[etailed], and diag[nostic]";
+        public const string FrameworkOptionDescription = "Target framework to publish for. The target framework has to be specified in the project file.";
+        public const string RuntimeOptionDescription = "Publish the project for a given runtime. This is used when creating self-contained deployment. Default is to publish a framework-dependent app.";
+        public const string ConfigurationOptionDescription = "Configuration to use for building the project.  Default for most projects is  \"Debug\".";
+        public const string CmdVersionSuffixDescription = "Defines the value for the $(VersionSuffix) property in the project.";
+
```
Comment (livarcocc) [inline src/dotnet/CommonLocalizableStrings.cs]: What's up with all the empty lines?

File: src/dotnet/CommonLocalizableStrings.cs (hunk 1 above)
Comment (wli3) [inline src/dotnet/CommonLocalizableStrings.cs]: If you look at the whole fine, it is organized by comment and sections. I think it is ok in this case. In general I try to reduce comments as much as I could, however, localization.cs file is more close to setting than actual code.

File: src/dotnet/CommonLocalizableStrings.cs (hunk 2 above)
Comment (wli3) [inline src/dotnet/CommonLocalizableStrings.cs]: Sorry about that, meanwhile I am still trying to make https://github.com/dotnet/codeformatter   work for this project

**Comment by wli3** [comment]
@dotnet-bot test RHEL7.2 x64 Release Build

**Comment by wli3** [comment]
@dotnet-bot test RHEL7.2 x64 Release Build

**Comment by livarcocc** [comment]
@dotnet-bot Test Ubuntu x64 Release Build
@dotnet-bot Test Ubuntu16.04 x64 Debug Build
@dotnet-bot Test Windows_NT x64 Release Build

**Comment by livarcocc** [comment]
@MattGertz for approval.

**Comment by wli3** [comment]
@livarcocc thanks for changing the target branch

**Pull request merged**